Core/Systems/IEcsDisposeSystem.cs

```diff
@@ -2,6 +2,6 @@
 {
 	public interface IEcsDisposeSystem : IEcsSystem
 	{
-		public void OnDispose();
+		public void OnOnDispose();
```

Comment:
let's just stick to OnDisponse... Creating a system (class) with method OnOnDispose seems like a misspelling and it's kinda confusing

Reply:
oops